Example/src/main/java/com/infomaniak/login/example/LoginActivity.kt

```diff
@@ -31,35 +33,46 @@ class LoginActivity : AppCompatActivity() {
                 lifecycleScope.launch {
                     val okHttpClient = OkHttpClient.Builder().build()
 
-                    infomaniakLogin.getToken(okHttpClient, it,
-                        { apiToken ->
-                            getTokenStatus.text = "${apiToken.userId} ${apiToken.accessToken}"
-
-                            btnDeconnect.apply {
-                                isVisible = true
-                                setOnClickListener { infomaniakLogin.deconnect(okHttpClient, apiToken) }
-                            }
-                        }, { error ->
-                            when (error) {
-                                InfomaniakLogin.ErrorStatus.SERVER -> getTokenStatus.text = "Server error"
-                                else -> getTokenStatus.text = "Error"
-                            }
-                        }
-                    )
+                    val tokenResult = infomaniakLogin.getToken(okHttpClient, code = it)
+                    Dispatchers.Main {
+                        updateUi(tokenResult, infomaniakLogin, okHttpClient)
+                    }
```

Comment:
This doesn't need to run on `Dipatchers.Main`, `lifecycleScope` already uses `Dispatchers.Main.immediate`.